American Whitewater/ViewControllers/Reaches/RunsListViewController.swift

```diff
@@ -248,6 +248,7 @@ class RunsListViewController: UIViewController {
 //        }
     
         self.refreshControl.beginRefreshingManually()
+        self.isLoadingData = true
```

Comment:
Just a thought, instead of creating a new variable that directly mirrors the refreshControl, can we just use the state of the refresh control?

Reply:
yeah totally agree